src/scalper.ts

```diff
@@ -642,12 +638,6 @@ Spot Δ: ${spotDelta} Offset Δ ${this.deltaOffset} Fair Value: ${fairValue}`,
       dipTotalGamma * stdDevSpread * fairValue * gammaThreshold,
       this.minSpotSize,
     );
-    if (Math.abs(hedgeDeltaTotal * fairValue) < (deltaThreshold * fairValue)) {
```

Comment:
Why remove? This is actually a slightly different case than 664. This is we're delta neutral and happy. Below is the instrument if we swept the order book at that price level we would be delta neutral. Duplicate but imo it's good info to have and treat seperate

Reply:
I was assuming that there never is a case where this would return but the later one wouldnt. There is no state change in between, so it would be redundant. If you think that info is necessary for the logging, can leave it. It is just logging a constant though.

Reply:
Good info to have when dealing with illiquid tokens. Let's leave

Reply:
Put back
That is the type of log spam that I will be removing in other places though.